specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2021-09-01-preview/appplatform.json

```diff
@@ -3458,6 +3666,10 @@
           "$ref": "#/definitions/PersistentDisk",
           "description": "Persistent disk settings"
         },
+        "customPersistentVolumes": {
```

Comment:
Shall we change the term from "volume" to "disk", to keep consistent w/ existing "persistentDisk" property?

Reply:
Changed to customPersistentDisk instead